src/attached_probe.cpp

```diff
@@ -960,11 +962,39 @@ void AttachedProbe::attach_kprobe(bool safe_mode)
     return;
   }
 
+  // Construct a string containing "module:function."
+  // Also throw a warning or error if the module doesn't exist,
+  // before attempting to attach.
+  // Note that we do not pass vmlinux, if it is specified.
+  std::string funcname = probe_.attach_point;
+  std::string modname = probe_.path;
+  if ((modname.length() > 0) && (modname.compare("vmlinux") != 0))
```

Comment:
Any reason not to do this?
```suggestion
  if ((modname.length() > 0) && modname != "vmlinux"))
```